sympy/printing/pretty/pretty.py

```diff
@@ -1522,6 +1521,30 @@ def _print_Contains(self, e):
         else:
             return prettyForm(sstr(e))
 
+    def _print_SeqFormula(self, s):
```

Comment:
That will cause issues

```
In [1]: from sympy.series.sequences import SeqExpr

In [2]: s = SeqExpr(n**2)

In [3]: s
Out[3]: ---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-3-f4d5d0c0671b> in <module>()
----> 1 s

/home/leosartaj/projects/sympy/my/lib/python2.7/site-packages/IPython/core/displayhook.pyc in __call__(self, result)
    251             self.write_output_prompt()
    252             format_dict, md_dict = self.compute_format_data(result)
--> 253             self.write_format_data(format_dict, md_dict)
    254             self.update_user_ns(result)
    255             self.log_output(format_dict)

/home/leosartaj/projects/sympy/my/lib/python2.7/site-packages/IPython/core/displayhook.pyc in write_format_data(self, format_dict, md_dict)
    172         # newline, even if all the prompt separators are ''. This is the
    173         # standard IPython behavior.
--> 174         result_repr = format_dict['text/plain']
    175         if '\n' in result_repr:
    176             # So that multi-line strings line up with the left column of

KeyError: 'text/plain'
```

Well that's because it is abstract class, and doesn't have all the methods. So, I think I will need to define printers explicitly.